javascripts/discourse/templates/components/d-modal.hbs

```diff
@@ -0,0 +1,46 @@
+<div class="modal-outer-container">
```

Comment:
I have found useful to have links that redirects to the Discourse code. In this case, a comment with the link of https://github.com/discourse/discourse/blob/master/app/assets/javascripts/discourse/app/templates/components/d-modal.hbs will allow any devs to quickly visit the Discourse current version and update if needed